_posts/2015-03-16-aerogear-ios-2.2.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: AeroGear iOS SDK v2.2 is out !
+---
+
+Hello AeroGear Community,
+
+The iOS team is happy to announce the next versions of AeroGear iOS libraries. Here is what's inside:
+
+**1. Synchronisation *(new)*:**
+
+With the [1.0.0-alpha1 sync release](/sync), AeroGear add a new feature to its tools belt: synchronization of data. Sync is declined server side: with Java and JavaScript libraries and client side: with Android and iOS platforms. To learn more about synchronisation, [read the sync announcement](/news/2015/02/17/aerogear-sync/index.html).
+
+The work around synchronisation on iOS is covered with:
+
+- [aerogear-ios-sync](https://github.com/aerogear/aerogear-ios-sync): the sync engine is where the algorithm is implemented. The sync algorithm is based on Google's [Differential Synchronisation](http://research.google.com/pubs/pub35605.html). We provide two implementations: one text-based [DiffMatchPatch](https://github.com/aerogear/aerogear-diffmatchpatch-ios) and a JsonPatch version based on [JSON Patch RFC6902](https://tools.ietf.org/html/rfc6902). 
+
+- [aerogear-ios-sync-client](https://github.com/aerogear/aerogear-ios-sync-client): the client uses sync engine and a network implementation. Our libraries is based on WebSockets to provide real-time sync.
+
+- [Jedi demo](https://github.com/aerogear/aerogear-ios-cookbook/tree/master/Jedi): Star war fan, rejoice in discovering our two Jedi hobbies demos (DiffMatchPatch and JsonPatch).
+      
+**2. House cleaning tasks:**
+
+- Move to [cocoapods 0.36.0](http://blog.cocoapods.org/CocoaPods-0.36/) for using iOS dynamic framework.
+
+- Deprecate [aerogear-ios-httpstub](https://github.com/aerogear/aerogear-ios-httpstub).
```

Comment:
let's add some more context here, deprecated in fav. of OHHTTPSsub + a link or something?